frontend/ui/elements/page_turns.lua

```diff
@@ -78,4 +97,48 @@ if Device:hasKeys() then
         end,
     })
 end
+
+local function genTapZonesMenu(tap_zones_type)
+    table.insert(PageTurns.sub_item_table[3].sub_item_table, {
+        text = tap_zones[tap_zones_type],
+        checked_func = function()
+            return G_reader_settings:readSetting("page_turns_tap_zones", "default") == tap_zones_type
+        end,
+        callback = function()
+            G_reader_settings:saveSetting("page_turns_tap_zones", tap_zones_type)
+            ReaderView:setupTouchZones()
+        end,
+    })
+end
+genTapZonesMenu("default")
+genTapZonesMenu("left_right")
+genTapZonesMenu("top_bottom")
+table.insert(PageTurns.sub_item_table[3].sub_item_table, {
+    text_func = function()
+        local size = math.floor(G_reader_settings:readSetting("page_turns_tap_zone_forward_size", DTAP_ZONE_FORWARD.w) * 100)
+        return T(_("Forward tap zone size: %1%"), size)
+    end,
+    enabled_func = function()
+        return G_reader_settings:readSetting("page_turns_tap_zones", "default") ~= "default"
+    end,
+    keep_menu_open = true,
+    callback = function(touchmenu_instance)
+        local is_left_right = G_reader_settings:readSetting("page_turns_tap_zones") == "left_right"
+        local size = math.floor(G_reader_settings:readSetting("page_turns_tap_zone_forward_size", DTAP_ZONE_FORWARD.w) * 100)
+        UIManager:show(require("ui/widget/spinwidget"):new{
+            title_text = is_left_right and _("Forward tap zone width") or _("Forward tap zone height"),
+            info_text = is_left_right and _("In percentage of screen width") or _("In percentage of screen height"),
```

Comment:
Wouldn't be a full sentence be better here?

Reply:
```
Forward tap zone width in percentage of screen width
```
What to add?

Reply:
Select forward tap zone width in percentage ...
or Enter, choose or another verb.

A quick glance shows that we mostly use sentences in info_text.

Reply:
I like the info-text as a clarification only, not too much verbose, when the action is obvious.

![08](https://user-images.githubusercontent.com/62179190/143019967-d6546fa2-53a5-47a4-867f-6773e1cd07a9.png)

Reply:
Well for me this is a sentence with an omittet (obvious) predicate. It has a period at the end ;)

Lets wait for our English teachers.

Reply:
I'm with @hius07, no need to be verbose. :-)

In context I think it'd look better to simply say:

```
Forward tap zone width
Percentage of screen width
```

By adding that `in` it becomes a bit weird (because it turns it into a full phrase split in two)